pkg/cmd/util/cacert/bsl_cacert.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cacert
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	kbclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	velerov1api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+)
+
+// GetCACertFromBackup fetches the BackupStorageLocation for a backup and returns its cacert
+func GetCACertFromBackup(ctx context.Context, client kbclient.Client, namespace string, backup *velerov1api.Backup) (string, error) {
```

Comment:
Sorry for the late review but it seems `GetCACertFromBackup` can call `GetCACertFromBSL`?

Reply:
yes it can.

Reply:
I swear I pushed that change... 👁️ _ 👁️

Reply:
https://github.com/vmware-tanzu/velero/compare/ba6865fdf3f550b45c3a9475af4bb6132214e9f3..b0fd2149b34012ee92b8bf919d90fbeb89d7d0b8

seems to be a mistake during rebase.